models/user.py

```diff
@@ -0,0 +1,11 @@
+from beanie import Document
+from beanie import StringProperty
+
+class UserModel(Document):
+    username = StringProperty(required=True)
+    password = StringProperty(required=True)
+    email = StringProperty(required=True)
+
```

Comment:
asynchronous Python object-document mapper (ODM) for MongoDB. Data models are based on [Pydantic](https://pydantic-docs.helpmanual.io/).

so you can use pydantic to your advantage pydantic is an uncle of typescript it makes typing restrictions on your code for healther coding avoiding bugs from production..

so you can do a preitter code like:
username: str  
password: str
email: EmailStr

Reply:
done i think